convert-string-to-camel/train.js

```diff
@@ -0,0 +1,27 @@
+function reachDestination(distance, speed) {
+  let returnValue;
+     let str = (distance / speed).toString();
+     let decimalPoint = str.indexOf('.');
+     if (decimalPoint == -1) {
+         returnValue = str;
+     }
+     else {
```

Comment:
This is a lot of logic for rounding! Note that there's a `Math.round` function that might be helpful in this case.